src/sections/trade/sections/swap/SwapPage.tsx

```diff
@@ -91,17 +96,32 @@ export function SwapPage() {
     )
   }
 
+  const assetInDefault =
```

Comment:
To keep the app consistent, I think we need similar defaults for the DCA trade page too.

Reply:
resolved